types/auth0/index.d.ts

```diff
@@ -2007,6 +2054,12 @@ export class ManagementClient<A = AppMetadata, U = UserMetadata> {
     deleteGuardianEnrollment(params: ObjectWithId): Promise<void>;
     deleteGuardianEnrollment(params: ObjectWithId, cb?: (err: Error) => void): void;
 
+    createGuardianEnrollmentTicket(data: sendEnrollmentTicketData): Promise<sendEnrollmentTicketResponse>;
+    createGuardianEnrollmentTicket(
+        data: sendEnrollmentTicketData,
+        cb?: (err: Error, data: sendEnrollmentTicketResponse) => void,
```

Comment:
```suggestion
        cb: (err: Error, data: sendEnrollmentTicketResponse) => void,
```
this probably should not be optional

Reply:
Thank you Peter, changes made